### ISSUE TITLE
 pool-eu-001-anycast.mc.tritondigital.com is breaking Podcasts

### ISSUE THREAD

**Issue by yoshimo** [mtxadmin/ublock#77]
This is used as a cname for Podcasts like
```derspiegel.mc.tritondigital.com``` and you can't get any episodes on your device without whitelisting it.
Other subdomains and the main domain should be checked for further false positives.

**Comment by mtxadmin** [comment]
Yes, seems it's a false positive, thanks. Though they have "Audio Advertising", "Audience Measurement", "Programmatic for Buyers" and this domain was in many lists, but it has been removed almost from everywhere.

Also, it seems they have tritondigitalmedia.com, it has been removed too.

Source list was updated, hosts files will be updated soon.
